install_linux.sh

```diff
@@ -4,6 +4,8 @@ processor=$(uname -m)
 
 if [ "$processor" == "x86_64" ]; then
   arch="amd64"
+elif [ "$processor" == "arm64" ]; then
```

Comment:
If you could submit this separately happy to merge it right away

Reply:
:+1: https://github.com/terraform-linters/tflint/pull/1185